pkg/docker/config/config.go

```diff
@@ -72,6 +72,9 @@ func SetAuthentication(sys *types.SystemContext, registry, username, password st
 			}
 			logrus.Debugf("failed to authenticate with the kernel keyring, falling back to authfiles. %v", err)
 		}
+		fmt.Fprintf(os.Stderr, "%s\n", `WARNING! Your password will be stored unencrypted in authentication file
```

Comment:
Hard-coding `os.Stderr` in a library is problematic. The closest we have for similar cases so far is `logrus.Warn*`; would that work here?

Reply:
Can I use the logrus.Warn* explicitly printout the warning without using --log-level?

Reply:
The default level in Logrus is “info” (relevant for Skopeo), and both Buildah and Podman default to “error”, so that should work fine AFAICS.

(One downside is that it uses the Logrus format, with some visual noise. That’s something to decide/configure at the top-level executable level, though.)